Chart.mqh

```diff
@@ -824,28 +824,28 @@ class Chart : public Market {
      * Returns bar's candle size in pips.
      */
     double GetBarCandleSize(uint _bar) {
-      return (GetClose(_bar) - GetOpen(_bar)) / GetPointsPerPip();
+      return (GetClose((int)_bar)- GetOpen(_bar)) / GetPointsPerPip();
     }
 
     /**
      * Returns bar's body size in pips.
      */
     double GetBarBodySize(uint _bar) {
-      return fabs(GetClose(_bar) - GetOpen(_bar)) / GetPointsPerPip();
+      return fabs(Chart::GetClose((int)_bar) - Chart::GetOpen((uint)_bar)) / GetPointsPerPip();
     }
 
     /**
      * Returns bar's head size in pips.
      */
     double GetBarHeadSize(uint _bar) {
-      return (GetHigh(_bar) - fmax(GetClose(_bar), GetOpen(_bar))) / GetPointsPerPip();
+      return (GetHigh(_bar) - fmax(Chart::GetClose((int)_bar), Chart::GetOpen((uint)_bar))) / GetPointsPerPip();
```

Comment:
Can we use consistent casting, either `uint` or `int`? Same below.

Reply:
Fixed